hack/test/unit

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#
+# Run unit tests
+#
+# TESTFLAGS - add additional test flags. Ex:
+#
+#   TESTFLAGS="-v -run TestBuild" hack/test/unit
+#
+# TESTDIRS - run tests for specified packages. Ex:
+#
+#    TESTDIRS="./pkg/term" hack/test/unit
+#
+set -eu -o pipefail
+
+TESTFLAGS+=" -test.timeout=${TIMEOUT:-5m}"
+BUILDFLAGS=( -tags "netgo seccomp libdm_no_deferred_remove" )
```

Comment:
Looks like I missed this; this changes the unit-tests to no longer use the `BUILDFLAGS` that were passed as env-var (and also changes them to be dynamically linked (previously statically))

Reply:
Not using BUILDFLAGS was the intent. Those are the flags for building a binary, not necessarily running tests.

Does that cause a problem?

Reply:
Caused me some digging why a backport didn't work, but worked on master 😂 

Perhaps we should rename the variable here, to prevent it from being confused with the `BUILDFLAGS` in the Makefile and so on 🤔